tools/integration_tests/mounting/main_test.go

```diff
@@ -48,24 +47,33 @@ func TestMain(m *testing.M) {
 		}
 	}
 
-	// Set up a directory into which we will build.
-	gBuildDir, err = ioutil.TempDir("", "gcsfuse_integration_tests")
-	if err != nil {
-		log.Fatalf("TempDir: %p", err)
-		return
-	}
-
 	// Build into that directory.
-	err = util.BuildGcsfuse(gBuildDir)
-	if err != nil {
-		log.Fatalf("buildGcsfuse: %p", err)
-		return
+	if !setup.TestInstalledPackage() {
```

Comment:
instead of bringing more code under if condition and using two if conditions, is it better to do something like:
if setup.TestInstalledPackage() {
    gBuildDir = "/"
    code := m.run()
    os.Exit(code)
}